experiment_helpers/gen_cbPermutation.m

```diff
@@ -8,48 +8,57 @@
 %conds: what conditions are being counterbalanced
     %this could include words (e.g. {'bead', 'bad', 'bed'})
     %or group assignment order (e.g. {'control', 'shifted'})
+    
+    % if conds is a cell array with multiple rows,
+    % e.g. {'a' 'a'; 'a' 'b'; 'a' 'c'; 'a' 'd'}
+    % then `conds` is considered to be the complete set of permutations
 %population: name of separate populations (e.g. 'control' or 'clinical')
     %or you can use this field if you want separate tracking for groups (i.e. 'control' or 'perturbed') in the study
 %%
 if nargin <1 || isempty(dataPath)
-    dataPath = ['\\wcs-cifs\wc\smng\experiments\' exptName];
+    dataPath = fullfile(get_exptLoadPath, exptName);
```

Comment:
Suggest calling this variable `savePath` for transparency

Reply:
Changed to permSavePath. See other note on unifying variable names.